### ISSUE TITLE
Fails to install on Mac

### ISSUE THREAD

**Issue by HDembinski** [impy-project/impy#4]
Following the instructions in the README, I tried to build on OSX, which failed:
```
CC=gcc-10 CXX=gcc-10 FC=gfortran-10 PYTHON_EXE=/usr/local/opt/python@3.8/bin/python3 make -j8
mkdir -p "/Users/hdembinski/Extern/impy/impy/lib"
/Library/Developer/CommandLineTools/usr/bin/make -C src
/Library/Developer/CommandLineTools/usr/bin/make -C EPOS-LHC install
/Library/Developer/CommandLineTools/usr/bin/make -C pythia6 install
/Library/Developer/CommandLineTools/usr/bin/make -C qgsjet install
/Library/Developer/CommandLineTools/usr/bin/make -C sophia install
/Library/Developer/CommandLineTools/usr/bin/make -C sibyll install
/Library/Developer/CommandLineTools/usr/bin/make -C dpmjet3.0-6 install
/Library/Developer/CommandLineTools/usr/bin/make -C dpmjetIII-19.1 install
/Library/Developer/CommandLineTools/usr/bin/make -C urqmd-3.4 install
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -m pythia6 -h pythia6.pyf --overwrite-signature \
	only: pyinit pyexec pytune pylist pyevnt pyevnw pystat pyedit pyhepc \
	pychge pycomp pyk impy_openlogfile impy_closelogfile : pythia-6.4.28.f /Users/hdembinski/Extern/impy/src/logging.f > pythia6.f2l 2>&1
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -m qgsII03 -h qgsII-03.pyf --overwrite-signature \
	only: cqgsini qgsect qgini qgconf qgreg chepevt impy_openlogfile impy_closelogfile : qgsjet-II-03.f \
	impy_qgsII.f /Users/hdembinski/Extern/impy/src/logging.f > qgsII03.f2l 2>&1
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -m eposlhc -h eposlhc.pyf --overwrite-signature only: aaset ainit aepos afinal hepmcstore getcharge idtrafo initializeepos initeposevt setstable setunstable xsection impy_openlogfile impy_closelogfile : sources/epos-bas-lhc.f sources/epos-ids-lhc.f sources/epos_interface.f /Users/hdembinski/Extern/impy/src/logging.f > epos-lhc.f2l 2>&1
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -m sophia -h sophia.pyf --overwrite-signature \
	only: eventgen print_event crossection impy_openlogfile impy_closelogfile : SOPHIA20.f eventgen.f sampling.f inpoutput.f jetset74dp.f /Users/hdembinski/Extern/impy/src/logging.f > sophia.f2l 2>&1
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -m dpmjet306 -h dpmjet306.pyf --overwrite-signature \
	only: pho_event dt_init dt_kkinc idt_icihad dt_xsglau pycomp dt_initjs dt_rndmst dt_rndm dt_inucas idt_ipdgha pho_init pho_setpar pho_pname pho_pmass pho_setmdl pho_setpdf pycomp pho_xsect pho_borncs pho_harmci pho_fitout pho_mcini pho_ptcut pytune pho_rregpar pho_sregpar pho_prevnt ipho_pdg2id ipho_id2pdg pho_harint pho_harxto pho_harxpt pho_setpcomb dt_phoxs dt_xshn dt_flahad impy_openlogfile impy_closelogfile : sources/dpmjet3.0-6.f sources/phojet1.12-35c4.f sources/pythia6115dpm3v1.f sources/user3.0-6.f /Users/hdembinski/Extern/impy/src/logging.f > dpmjet306.f2l 2>&1
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -m sib21 -h sib21.pyf --overwrite-signature only: sibyll sibyll_ini sib_sigma_hp sib_sigma_hair sib_sigma_hnuc int_nuc decsib decpar sibini sib_list isib_pid2pdg isib_pdg2pid pdg_ini impy_openlogfile impy_closelogfile sibhep1 : sibyll_21.f sib21aux.f  /Users/hdembinski/Extern/impy/src/sibyll/sibyll_init.f /Users/hdembinski/Extern/impy/src/logging.f > sib21.f2l 2>&1
make[2]: *** No rule to make target `install'.  Stop.
make[1]: *** [dpmjetIII-19.1] Error 2
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch pythia-6.4.28.f
make[1]: *** Waiting for unfinished jobs....
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -m urqmd34 -h urqmd34.pyf --overwrite-signature \
	only: urqmd init uinit set0 params uounit strini loginit loadwtab norm_init output cascinit nucrad urqini partname chepevt ptsigtot init_rmmard impy_openlogfile impy_closelogfile  : sources/1fluid.f sources/bessel.f sources/delpart.f sources/getmass.f sources/hepcmp.f sources/iso.f sources/numrec.f sources/pythia6409.f sources/siglookup.f sources/upmerge.f sources/addpart.f sources/blockres.f sources/coload.f sources/detbal.f sources/getspin.f sources/hepnam.f sources/ityp2pdg.f sources/output.f sources/string.f sources/urqmd.f sources/angdis.f sources/boxprg.f sources/dwidth.f sources/init.f sources/jdecay2.f sources/paulibl.f sources/saveinfo.f sources/tabinit.f sources/whichres.f sources/anndec.f sources/cascinit.f sources/dectim.f sources/error.f sources/hepchg.f sources/input.f sources/make22.f sources/proppot.f sources/scatter.f sources/uhmerge.f sources/urqinit.f sources/CFmax.f90 sources/quadri.f90 sources/cornelius.f90 \
	/Users/hdembinski/Extern/impy/src/logging.f /Users/hdembinski/Extern/impy/src/rangen.f impy_urqmd.f > urqmd34.f2l 2>&1
gfortran -c -cpp -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch sib21aux.f
gfortran -c -cpp -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch /Users/hdembinski/Extern/impy/src/sibyll/sibyll_init.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -cpp -DSIBYLL_SP ../rangen.f -o sib21rnd.o
gfortran -c -cpp -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch sibyll_21.f
sibyll_21.f:1703:116:

 1703 |       write(LUN,'(1x,a,i3,3x,a,i3))') 'Total charge:',ichar,
      |                                                                                                                    1
Warning: Extraneous characters in format at (1)
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch qgsjet-II-03.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch SOPHIA20.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch eventgen.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch sampling.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch inpoutput.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch jetset74dp.f
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -m sib23 -h sib23.pyf --overwrite-signature \
	only: sibyll sibyll_ini sib_sigma_hp sib_sigma_hair sib_sigma_hnuc int_nuc decsib decpar sibini sib_list isib_pid2pdg isib_pdg2pid pdg_ini impy_openlogfile impy_closelogfile sibhep3 : sibyll2.3.f /Users/hdembinski/Extern/impy/src/sibyll/sibyll_init.f /Users/hdembinski/Extern/impy/src/logging.f > sib23.f2l 2>&1
gfortran -c -cpp -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch sibyll2.3.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-con-lhc.o sources/epos-con-lhc.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-dky-lhc.o sources/epos-dky-lhc.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-dro-lhc.o sources/epos-dro-lhc.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -Wno-argument-mismatch -std=legacy  -o sources/dpmjet3.0-6.o sources/dpmjet3.0-6.f
sources/dpmjet3.0-6.f:16971:24:

16971 |             R = DT_RNDM(PCAS)
      |                        1
Warning: Rank mismatch in argument 'vdummy' at (1) (scalar and rank-1)
sources/dpmjet3.0-6.f:14277:21:

14277 |          IF (DT_RNDM(PT).GT.OHALF) THEN
      |                     1
Warning: Rank mismatch in argument 'vdummy' at (1) (scalar and rank-1)
sources/dpmjet3.0-6.f:14348:21:

14348 |          IF (DT_RNDM(PT).GT.OHALF) THEN
      |                     1
Warning: Rank mismatch in argument 'vdummy' at (1) (scalar and rank-1)
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch impy_qgsII.f
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -m qgsII04 -h qgsII-04.pyf --overwrite-signature \
	only: cqgsini qgsect qgini qgconf qgreg chepevt impy_openlogfile impy_closelogfile : qgsjet-II-04.f \
	impy_qgsII.f /Users/hdembinski/Extern/impy/src/logging.f > qgsII04.f2l 2>&1
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-ems-lhc.o sources/epos-ems-lhc.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch qgsjet-II-04.f
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -m sib23c00 -h sib23c00.pyf --overwrite-signature \
	only: sibyll sibyll_ini sib_sigma_hp sib_sigma_hair sib_sigma_hnuc int_nuc decsib decpar sibini sib_list isib_pid2pdg isib_pdg2pid pdg_ini impy_openlogfile impy_closelogfile sibhep3 : sibyll2.3c00.f /Users/hdembinski/Extern/impy/src/sibyll/sibyll_init.f /Users/hdembinski/Extern/impy/src/logging.f > sib23c00.f2l 2>&1
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -c --compiler=unix --fcompiler=gnu95 --opt="-O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch" sophia.pyf \
	/Users/hdembinski/Extern/impy/src/rangen.o /Users/hdembinski/Extern/impy/src/logging.o SOPHIA20.o eventgen.o sampling.o inpoutput.o jetset74dp.o >> sophia.f2l 2>&1
gfortran -c -cpp -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch sibyll2.3c00.f
make[2]: *** [sophia.cpython-38-darwin.so] Error 1
make[1]: *** [sophia] Error 2
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/1fluid.o sources/1fluid.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-fra-lhc.o sources/epos-fra-lhc.f
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -m sib23c01 -h sib23c01.pyf --overwrite-signature \
	only: sibyll sibyll_ini sib_sigma_hp sib_sigma_hair sib_sigma_hnuc int_nuc decsib decpar sibini sib_list isib_pid2pdg isib_pdg2pid pdg_ini impy_openlogfile impy_closelogfile sibhep3 : sibyll2.3c01.f /Users/hdembinski/Extern/impy/src/sibyll/sibyll_init.f /Users/hdembinski/Extern/impy/src/logging.f > sib23c01.f2l 2>&1
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -Wno-argument-mismatch -std=legacy  -o sources/phojet1.12-35c4.o sources/phojet1.12-35c4.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-hnb-lhc.o sources/epos-hnb-lhc.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/bessel.o sources/bessel.f
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -m qgs01 -h qgs01.pyf --overwrite-signature \
	only: cqgsini sectnu xxaini psconf xxreg psaini chepevt impy_openlogfile impy_closelogfile : qgsjet01d.f \
	impy_qgs1.f /Users/hdembinski/Extern/impy/src/logging.f > qgs01.f2l 2>&1
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -Wno-argument-mismatch -std=legacy  -o sources/pythia6115dpm3v1.o sources/pythia6115dpm3v1.f
gfortran -c -cpp -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch sibyll2.3c01.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-ico-lhc.o sources/epos-ico-lhc.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-ids-lhc.o sources/epos-ids-lhc.f
sources/pythia6115dpm3v1.f:9757:51:

 9757 |           CALL PYXDIF(NC,NJET,KFLC,P(ID,5),CHI,THE,PHI)
      |                                                   1
Warning: Rank mismatch in argument 'phi' at (1) (scalar and rank-1)
sources/pythia6115dpm3v1.f:9761:33:

 9761 |           CALL PYROBO(NC+1,N,THE,PHI,0D0,0D0,0D0)
      |                                 1
Warning: Rank mismatch in argument 'phi' at (1) (scalar and rank-1)
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch impy_qgs1.f
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -m sib23c02 -h sib23c02.pyf --overwrite-signature \
	only: sibyll sibyll_ini sib_sigma_hp sib_sigma_hair sib_sigma_hnuc int_nuc decsib decpar sibini sib_list isib_pid2pdg isib_pdg2pid pdg_ini impy_openlogfile impy_closelogfile sibhep3 : sibyll2.3c02.f /Users/hdembinski/Extern/impy/src/sibyll/sibyll_init.f /Users/hdembinski/Extern/impy/src/logging.f > sib23c02.f2l 2>&1
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/delpart.o sources/delpart.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/getmass.o sources/getmass.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/hepcmp.o sources/hepcmp.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/iso.o sources/iso.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -Wno-argument-mismatch -std=legacy  -o sources/user3.0-6.o sources/user3.0-6.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch qgsjet01d.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/numrec.o sources/numrec.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/pythia6409.o sources/pythia6409.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-int-lhc.o sources/epos-int-lhc.f
gfortran -c -cpp -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch sibyll2.3c02.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-jps-lhc.o sources/epos-jps-lhc.f
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -m sib23c03 -h sib23c03.pyf --overwrite-signature \
	only: sibyll sibyll_ini sib_sigma_hp sib_sigma_hair sib_sigma_hnuc int_nuc decsib decpar sibini sib_list isib_pid2pdg isib_pdg2pid pdg_ini impy_openlogfile impy_closelogfile sibhep3 : sibyll2.3c03.f /Users/hdembinski/Extern/impy/src/sibyll/sibyll_init.f /Users/hdembinski/Extern/impy/src/logging.f > sib23c03.f2l 2>&1
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-omg-lhc.o sources/epos-omg-lhc.f
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -c --compiler=unix --fcompiler=gnu95 --opt="-O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch" qgsII-03.pyf \
	qgsjet-II-03.o /Users/hdembinski/Extern/impy/src/rangen.o /Users/hdembinski/Extern/impy/src/logging.o impy_qgsII.o >> qgsII03.f2l 2>&1
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/siglookup.o sources/siglookup.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-par-lhc.o sources/epos-par-lhc.f
make[2]: *** [qgsII03.cpython-38-darwin.so] Error 1
make[1]: *** [qgsjet] Error 2
gfortran -c -cpp -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch sibyll2.3c03.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/upmerge.o sources/upmerge.f
sources/pythia6409.f:50940:31-41:

50940 |         WRITE(MSTU(11),*) (XXc(I),I=9,12)
      |                               1         2
Warning: Array reference at (1) out of bounds (12 > 10) in loop beginning at (2)
sources/pythia6409.f:50941:31-42:

50941 |         WRITE(MSTU(11),*) (XXc(I),I=13,16)
      |                               1          2
Warning: Array reference at (1) out of bounds (13 > 10) in loop beginning at (2)
sources/pythia6409.f:50941:31-42:

50941 |         WRITE(MSTU(11),*) (XXc(I),I=13,16)
      |                               1          2
Warning: Array reference at (1) out of bounds (16 > 10) in loop beginning at (2)
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/addpart.o sources/addpart.f
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -m sib23d -h sib23d.pyf --overwrite-signature \
	only: sibyll sibyll_ini sib_sigma_hp sib_sigma_hair sib_sigma_hnuc int_nuc decsib decpar sibini sib_list isib_pid2pdg isib_pdg2pid pdg_ini impy_openlogfile impy_closelogfile sibhep3 : sibyll2.3d.f /Users/hdembinski/Extern/impy/src/sibyll/sibyll_init.f /Users/hdembinski/Extern/impy/src/logging.f > sib23d.f2l 2>&1
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-qsh-lhc.o sources/epos-qsh-lhc.f
gfortran -c -cpp -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch sibyll2.3d.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/blockres.o sources/blockres.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-random.o sources/epos-random.f
sources/epos-random.f:50:19:

   50 |       call RMMARD( uni,1,irndmseq)
      |                   1
Warning: Rank mismatch in argument 'rvec' at (1) (rank-1 and scalar)
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -c --compiler=unix --fcompiler=gnu95 --opt="-O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch" sib21.pyf sibyll_21.o sib21aux.o /Users/hdembinski/Extern/impy/src/sibyll/sibyll_init.o /Users/hdembinski/Extern/impy/src/logging.o sib21rnd.o >> sib21.f2l 2>&1
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-rsh-lhc.o sources/epos-rsh-lhc.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-sem-lhc.o sources/epos-sem-lhc.f
make[2]: *** [sib21.cpython-38-darwin.so] Error 1
make[2]: *** Waiting for unfinished jobs....
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/coload.o sources/coload.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/detbal.o sources/detbal.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/getspin.o sources/getspin.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-tim-lhc.o sources/epos-tim-lhc.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-uti-lhc.o sources/epos-uti-lhc.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/hepnam.o sources/hepnam.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-xan-lhc.o sources/epos-xan-lhc.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/ityp2pdg.o sources/ityp2pdg.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos-xpr-lhc.o sources/epos-xpr-lhc.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/output.o sources/output.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos_interface.o sources/epos_interface.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/epos_j.o sources/epos_j.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/string.o sources/string.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/urqmd.o sources/urqmd.f
make[1]: *** [sibyll] Error 2
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/eposm.o sources/eposm.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/angdis.o sources/angdis.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/boxprg.o sources/boxprg.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/dwidth.o sources/dwidth.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/init.o sources/init.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/jdecay2.o sources/jdecay2.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/paulibl.o sources/paulibl.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/saveinfo.o sources/saveinfo.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/tabinit.o sources/tabinit.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/whichres.o sources/whichres.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/anndec.o sources/anndec.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/cascinit.o sources/cascinit.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/dectim.o sources/dectim.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/error.o sources/error.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/hepchg.o sources/hepchg.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/input.o sources/input.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/make22.o sources/make22.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/proppot.o sources/proppot.f
sources/make22.f:1644:45:

 1644 |          call getbran((/sig1,sig2,sig3/),1,3,dummy,1,3,channel)
      |                                             1
......
 3739 |       call getbran(x,minnuc,maxmes,xmax,im,ip,ir)
      |                                   2
Warning: Type mismatch between actual argument at (1) and actual argument at (2) (INTEGER(4)/REAL(8)).
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/scatter.o sources/scatter.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/uhmerge.o sources/uhmerge.f
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -o sources/urqinit.o sources/urqinit.f
gfortran -c -O3 -Wno-uninitialized -fPIC -fno-second-underscore -fallow-argument-mismatch -o sources/CFmax.o sources/CFmax.f90
gfortran -c -O3 -Wno-uninitialized -fPIC -fno-second-underscore -fallow-argument-mismatch -o sources/quadri.o sources/quadri.f90
gfortran -c -O3 -Wno-uninitialized -fPIC -fno-second-underscore -fallow-argument-mismatch -o sources/cornelius.o sources/cornelius.f90
gfortran -c -O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch impy_urqmd.f
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -c --compiler=unix --fcompiler=gnu95 --opt="-O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch" eposlhc.pyf sources/epos-bas-lhc.o sources/epos-con-lhc.o sources/epos-dky-lhc.o sources/epos-dro-lhc.o sources/epos-ems-lhc.o sources/epos-fra-lhc.o sources/epos-hnb-lhc.o sources/epos-ico-lhc.o sources/epos-ids-lhc.o sources/epos-int-lhc.o sources/epos-jps-lhc.o sources/epos-omg-lhc.o sources/epos-par-lhc.o sources/epos-qsh-lhc.o sources/epos-random.o sources/epos-rsh-lhc.o sources/epos-sem-lhc.o sources/epos-tim-lhc.o sources/epos-uti-lhc.o sources/epos-xan-lhc.o sources/epos-xpr-lhc.o sources/epos_interface.o sources/epos_j.o sources/eposm.o /Users/hdembinski/Extern/impy/src/logging.o >> epos-lhc.f2l 2>&1
make[2]: *** [eposlhc.cpython-38-darwin.so] Error 1
make[1]: *** [EPOS-LHC] Error 2
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -c --compiler=unix --fcompiler=gnu95 --opt="-O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch -Wno-argument-mismatch" \
	dpmjet306.pyf sources/dpmjet3.0-6.o sources/phojet1.12-35c4.o sources/pythia6115dpm3v1.o sources/user3.0-6.o /Users/hdembinski/Extern/impy/src/logging.o >> dpmjet306.f2l 2>&1
make[2]: *** [dpmjet306.cpython-38-darwin.so] Error 1
make[1]: *** [dpmjet3.0-6] Error 2
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -c --compiler=unix --fcompiler=gnu95 --opt="-O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch" pythia6.pyf \
	pythia-6.4.28.o /Users/hdembinski/Extern/impy/src/logging.o >> pythia6.f2l 2>&1
make[2]: *** [pythia6.cpython-38-darwin.so] Error 1
make[1]: *** [pythia6] Error 2
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -c --compiler=unix --fcompiler=gnu95 --opt="-O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch" urqmd34.pyf \
	/Users/hdembinski/Extern/impy/src/logging.o /Users/hdembinski/Extern/impy/src/rangen.o sources/1fluid.o sources/bessel.o sources/delpart.o sources/getmass.o sources/hepcmp.o sources/iso.o sources/numrec.o sources/pythia6409.o sources/siglookup.o sources/upmerge.o sources/addpart.o sources/blockres.o sources/coload.o sources/detbal.o sources/getspin.o sources/hepnam.o sources/ityp2pdg.o sources/output.o sources/string.o sources/urqmd.o sources/angdis.o sources/boxprg.o sources/dwidth.o sources/init.o sources/jdecay2.o sources/paulibl.o sources/saveinfo.o sources/tabinit.o sources/whichres.o sources/anndec.o sources/cascinit.o sources/dectim.o sources/error.o sources/hepchg.o sources/input.o sources/make22.o sources/proppot.o sources/scatter.o sources/uhmerge.o sources/urqinit.o sources/CFmax.o sources/quadri.o sources/cornelius.o impy_urqmd.o >> urqmd34.f2l 2>&1
make[2]: *** [urqmd34.cpython-38-darwin.so] Error 1
make[1]: *** [urqmd-3.4] Error 2
make: *** [src] Error 2
```
More precisely, there seems to occur here:
```
CC=gcc-10 CXX=gcc-10 FC=gfortran-10 PYTHON_EXE=/usr/local/opt/python@3.8/bin/python3 make -j1
mkdir -p "/Users/hdembinski/Extern/impy/impy/lib"
/Library/Developer/CommandLineTools/usr/bin/make -C src
/Library/Developer/CommandLineTools/usr/bin/make -C EPOS-LHC install
/usr/local/opt/python@3.8/bin/python3  -m numpy.f2py --quiet -c --compiler=unix --fcompiler=gnu95 --opt="-O3 -Wno-uninitialized -fPIC -std=legacy -fallow-argument-mismatch" eposlhc.pyf sources/epos-bas-lhc.o sources/epos-con-lhc.o sources/epos-dky-lhc.o sources/epos-dro-lhc.o sources/epos-ems-lhc.o sources/epos-fra-lhc.o sources/epos-hnb-lhc.o sources/epos-ico-lhc.o sources/epos-ids-lhc.o sources/epos-int-lhc.o sources/epos-jps-lhc.o sources/epos-omg-lhc.o sources/epos-par-lhc.o sources/epos-qsh-lhc.o sources/epos-random.o sources/epos-rsh-lhc.o sources/epos-sem-lhc.o sources/epos-tim-lhc.o sources/epos-uti-lhc.o sources/epos-xan-lhc.o sources/epos-xpr-lhc.o sources/epos_interface.o sources/epos_j.o sources/eposm.o /Users/hdembinski/Extern/impy/src/logging.o >> epos-lhc.f2l 2>&1
make[2]: *** [eposlhc.cpython-38-darwin.so] Error 1
make[1]: *** [EPOS-LHC] Error 2
make: *** [src] Error 2
```

**Comment by afedynitch** [comment]
Can you attach the src/EPOSLHC/epos-lhc.f2l or post the contents?

**Comment by HDembinski** [comment]
fixed by #39
